spython/instance/cmd/start.py

```diff
@@ -14,9 +14,11 @@ def start(
     name=None,
     args=None,
     sudo=False,
+    sudo_options=None,
     options=None,
     capture=False,
     singularity_options=None,
+    env=None,
```

Comment:
Let's use `environ` to match `os.environ`, also `env` is by default defined in Ipython sessions, so we wouldn't want to override it.